src/gds/ua_registration_manager.h

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2018 (c) Markus Karch, Fraunhofer IOSB
+ */
+
+
+#ifndef OPEN62541_UA_REGISTRATION_MANAGER_H
+#define OPEN62541_UA_REGISTRATION_MANAGER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "ua_plugin_log.h"
+#include "ua_server.h"
+#include "ua_record_datatype.h"
+#include "open62541_queue.h"
```

Comment:
Are all these includes really required here? E.g. log?

Try to use forward declarations if possible and suitable.